nitro-node/test/model.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
What is the purpose of this mock file? I'm seeing that the test settings will need only 2 values: model's URL & template.

Reply:
This json file is copied from Jan downloaded model directories for a quick test on the implementation, will remove unnecessary config then.